examples/pi-hole/docker-compose.yml

```diff
@@ -29,7 +29,7 @@ services:
       default:
         ipv4_address: 172.28.0.2
     healthcheck:
-      test: ["CMD", "drill", "@127.0.0.1", "dnssec.works"]
+      test: ["CMD", "drill", "@127.0.0.1", "dnssec.works", "||", "exit", "1"]
```

Comment:
Unfortunately the `||` operator only exists in shell commands (e.g. `sh`, `bash`, etc) and this image has none of those. Will need to consider a different approach to solve this.

In any non-distroless container you could do something like the following:
```yml
test: ["CMD", "sh", "-c", "drill @127.0.0.1 dnssec.works || exit 1"]
```
But not here I'm afraid